clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -170,6 +187,26 @@ static bool omitRegionTerm(mlir::Region &r) {
   return singleNonEmptyBlock && yieldsNothing();
 }
 
+void printVisibilityAttr(OpAsmPrinter &printer,
+                         cir::VisibilityAttr &visibility) {
+  switch (visibility.getValue()) {
+  case cir::VisibilityKind::Hidden:
+    printer << "hidden";
+    break;
+  case cir::VisibilityKind::Protected:
```

Comment:
What other values here are there?  Can we skip the `default` here and just have it try to print all?

Reply:
VisibilityKind can be `hidden`, `protected`, or `default`, so I don't think there is any value in printing `default`. I can make it explicit in the switch and remove the default case though.